unix/mpconfigport.h

```diff
@@ -96,6 +96,7 @@
 #define MICROPY_PY_SYS_MAXSIZE      (1)
 #define MICROPY_PY_SYS_STDFILES     (1)
 #define MICROPY_PY_SYS_EXC_INFO     (1)
+#define MICROPY_PY_SYS_GETSIZEOF    (0)
```

Comment:
For config options that are left at their default value (in this case disabled) I prefer to leave them out of the config file.